.envrc

```diff
@@ -0,0 +1,8 @@
+nix_direnv_watch_file default.nix
+nix_direnv_watch_file shell.nix
+for f in $(find dist/nix/ -type f); do
+	nix_direnv_watch_file $f
+done
+#use nix
+#use flake '.?submodules=1'
+use flake
```

Comment:
We try to avoid adding such files to the repository. It is preferable to add a `dist/nix/README.md` and add it there as a recommended `.envrc` people wishing to use nix can use (and you can add it to `.gitignore` so those users are not annoyed by the untracked file if they do).
Actually, such a README.md would be desirable in general, describing how to use the nix infrastructure in a few sentences, linking to more documentation for those who want to dig deeper.